### PR TITLE
Fix hotkeys for Multi images (Compound files)

### PR DESCRIPTION
**Pull request by sergzak022**
## Description
Fix a bug with hotkeys that happens in compound file context with multiple images. Updated HotkeyListener documentation to make it easier to use.

Bug Description:
Pressing a hotkey makes registered hotkey callbacks fire for all the image annotation panes

### CODE REVIEW

File: frontend/src/components/annotation/image_and_video_annotation/annotation_core.vue [hunk 1]
```diff
@@ -1733,6 +1741,10 @@ export default Vue.extend({
 
     this.setupHotkeys(this.hotkeyListenerScope)
 
+    if ( this.is_active ) {
```
Comment (sergzak022) [inline frontend/src/components/annotation/image_and_video_annotation/annotation_core.vue:1744]: Once image annotation components are mounted, need to activate registered hotkeys ofr the active one.

File: frontend/src/components/annotation/image_and_video_annotation/annotation_core.vue [hunk 2]
```diff
@@ -741,6 +741,14 @@ export default Vue.extend({
   watch: {
     is_active: function (){
       this.canvas_element.style.cursor = ''
+      if ( !this.hotkeyListenerScope ) {
+        return
+      }
+      if ( this.is_active) {
+        this.hotkey_listener.setScopes([this.hotkeyListenerScope])
+      } else {
```
Comment (sergzak022) [inline frontend/src/components/annotation/image_and_video_annotation/annotation_core.vue:749]: The else block is needed for situations when user annotates in the compound context. For example text_annotation component currently uses a different mechanism for registering hotkeys. If user switches from image annotation to text in the same compound context, we needs to deactivate image annotation hotkeys.

**Comment by sergzak022** [comment]
Hey Anthony, I'm not removing old hotkey manager class because it looks like it's still used by text_annotation_core component. Please see `listeners_map` method of `src/components/annotation/annotation_ui_factory.vue` and `keydown_event_listeners`/`keyup_event_listeners` methods of `text_annotation_core.vue`. Once we move `text_annotation_core` to use the new HotkeyListener, we can get rid of HotkeyManager.

**Comment by anthony-sarkis** [comment]
Makes sense regarding keeping legacy `HotkeyManager` until text moves over to new one.

Tested manually and appears all working as expected now, thanks Sergey!

**Comment by anthony-sarkis** [comment]
One small note for future is when we do remove legacy `HotkeyManager`, I believe we want to move the `is_active` scope setting to the higher level component (e.g. where `HotkeyManager` is currently sitting) to minimize code in "leaf" components.

**Review by anthony-sarkis — APPROVED** [review]

**Pull request merged**